lib/lunchbot_web/live/component/shared_cart/shared_cart.html.heex

```diff
@@ -0,0 +1,16 @@
+<div id="shared-cart">
+
+  <div>
+    <.form let={f} for={@changeset} id="attempt2" phx-change="change-office">
+        <%= label f, :office_selection %>
+        <%= Phoenix.HTML.Form.select f, :office_selection, Enum.map(@offices, &{&1.name, &1.id}) %>
+    </.form>
+  </div>
+
+  <h2>Shared Cart</h2>
```

Comment:
Really neat logic here

Reply:
Yeah honestly I was shocked when that syntax worked as I expected it with the `.id` `.name` getting used exactly as I thought they would lol